modules/caddyhttp/vars.go

```diff
@@ -88,6 +89,74 @@ func (m VarsMatcher) Match(r *http.Request) bool {
 	return true
 }
 
+// MatchVarRE matches the value of the context variables by a given regular expression.
+//
+// Upon a match, it adds placeholders to the request: `{http.regexp.name.capture_group}`
+// where `name` is the regular expression's name, and `capture_group` is either
+// the named or positional capture group from the expression itself. If no name
+// is given, then the placeholder omits the name: `{http.regexp.capture_group}`
+// (potentially leading to collisions).
+type MatchVarRE map[string]*MatchRegexp
```

Comment:
```suggestion
type MatchVarsRE map[string]*MatchRegexp
```

For consistency with `VarsMiddleware`